src/main/java/dansplugins/factionsystem/utils/Locale.java

```diff
@@ -0,0 +1,13 @@
+package dansplugins.factionsystem.utils;
+
+import dansplugins.factionsystem.services.LocalLocaleService;
+
+/**
+ * @author Daniel McCoy Stephenson
+ */
+public class Locale {
+    
+    public static String get(String key) {
```

Comment:
It's still the same as if you left static in the file LocalLocaleService 🤔 , I think the problem is elsewhere, I will check it again after I get back from school

Reply:
This is just intended to make it easier to utilize the LocalLocaleService 👍 If we move to another solution it may be rendered unnecessary.